### ISSUE TITLE
Toggling real-time mode does not update current state (Trac #19)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#23]
When the real-time mode button is clicked, the state does not actually update until the step number changes.

Migrated from http://microfluidics.utoronto.ca/ticket/19
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "When the real-time mode button is clicked, the state does not actually update until the step number changes.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Toggling real-time mode does not update current state", 
    "priority": "minor", 
    "keywords": "real-time mode", 
    "version": "0.1", 
    "time": "2011-10-14T16:57:33", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120828T16:34:08: 
* ryan@fobel.net commented:

> commit 4fc51e110cd0e1273116f872b3ffe4b2e2c6fde9
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Tue Aug 28 12:25:25 2012 -0400
> 
>     Fixes #23 (Toggling real-time mode does not...
>     
>     Fixes #23 (Toggling real-time mode does not update current state).


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "gui" to "microdrop"
